install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -494,20 +494,17 @@ data:
   {{- if ne (.Values.routingMode | default "tunnel") "tunnel" }}
     {{- fail (printf "RoutingMode must be set to tunnel when aksbyocni.enabled=true" )}}
   {{- end }}
-  routing-mode: "tunnel"
-  tunnel-protocol: "vxlan"
-{{- else if .Values.routingMode }}
-  routing-mode: {{ .Values.routingMode | quote }}
-{{- else }}
-  # Default case
-  routing-mode: "tunnel"
-  tunnel-protocol: "vxlan"
+  {{- if ne (.Values.tunnelProtocol | default "vxlan" ) "vxlan" }}
+    {{- fail (printf "TunnelProtocol must be set to vxlan when aksbyocni.enabled=true" )}}
+  {{- end }}
```

Comment:
I wonder if this is an actual constraint, or any tunnel protocol would work correctly.

Reply:
Yeah let's drop that check and just ensure that we're running in any tunnel mode. I don't see any indication in the code that it only supports VXLAN. 

It is possible that this check is a leftover from where `routing-mode` and `tunnel-protocol` where one single flag called `tunnel`.

Reply:
Dropped and added the reference to the split of settings, as you mentioned it. If this is incorrect or misleading, I will remove this.

Reply:
Looks good! Thanks for adding the release note